includes/class-options.php

```diff
@@ -30,6 +28,16 @@ public static function init() {
 		\add_filter( 'option_activitypub_max_image_attachments', array( self::class, 'default_max_image_attachments' ) );
 	}
 
+	/**
+	 * Delete all options.
+	 */
+	public static function delete() {
+		global $wpdb;
+		// phpcs:ignore WordPress.DB.DirectDatabaseQuery
+		$wpdb->query( // phpcs:ignore WordPress.DB.DirectDatabaseQuery
+			"DELETE FROM $wpdb->options WHERE option_name LIKE 'activitypub_%'"
+		);
```

Comment:
```suggestion

		// phpcs:ignore WordPress.DB.DirectDatabaseQuery
		$wpdb->query( "DELETE FROM $wpdb->options WHERE option_name LIKE 'activitypub_%'" );
```
One ignore should probably be sufficient?